src/interfaces/invitation.ts

```diff
@@ -0,0 +1,13 @@
+import { PermissionLevel } from 'graasp';
+
+interface Invitation {
```

Comment:
I cannot set `id` or `createdAt` in `BaseInvitation` since they are created on insert. It would be really nice to link them both, does it make sense to have `implements Partial<Invitation>`?

Reply:
I think it makes sense, if the compiler is able to infer which fields actually exist (or not) at usage site. I wrote a [toy example, it seems to work](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgCoQM5mQbwFDKHJwBcyWUoA5gUQEZkgCuAtndLYQmQPZ0BWEBGDwBfPHgQAbOBgzIAwjxAIoESMmAsADlIgsI4eQAU4UMMDhSAPOiwA+XJ2JkK1Zw2TM2HIsgTKFEzCPFAAFKTkYJQgVAA0yJ7e7FAAlE5+fmAAFsAYAHRwyAC8xM5ZuQV0JYnO4uKSgdgINSAQAO6KyqrqEGEARDA8PP0JACwATKkA3JKFc3RAA) with the expected behaviour for nullables.